aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/SigV4SigningDecorator.kt

```diff
@@ -120,6 +120,19 @@ class SigV4SigningConfig(
                     )
                 }
             }
+            is ServiceConfig.OperationConfigOverride -> {
+                if (runtimeMode.generateOrchestrator) {
+                    rustTemplate(
+                        """
+                        resolver.config_mut()
```

Comment:
We should probably only do this if the SigningRegion isn't already set, since this could unintentionally overwrite another runtime plugin that set it.

Reply:
In this case, customers intentionally override
```
client
    .operation()
    .customize()
    .await
    .unwrap()
    .config_override(Config::builder().region(Region::new("us-east-1")))
    ... // and so on
```
Isn't this a surprise to them if `us-east-1` is not respected? Or are we saying they specify a `Region` from a different mechanism other than `config_override`?

Reply:
Oh, is this only loading from the config override, and not from a combination of the client config and override config? If that's the case, then this should be fine.

Reply:
Ah, I see what you mean. Yes, this is prior to combining the client config and override config.